samples/subsys/fs/littlefs/boards/stm32h747i_disco_m7.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=8192
+CONFIG_SHELL=y
+CONFIG_FILE_SYSTEM_SHELL=y
+CONFIG_SDMMC_STM32_HWFC=y
+CONFIG_SHELL_STACK_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_COMMON_LIBC_MALLOC=y
```

Comment:
I wonder why this option would be required specifically with this board.
Can you check if it's strictly required ?

Reply:
@erwango  I have done some experiments on this. All of these are not required. But following is needed

1. CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=2048 - without this I get  fs mount error
*** Booting Zephyr OS build zephyr-v3.5.0-500-g57dd116d8b32 ***
Sample program to r/w files on littlefs
I: LittleFS version 2.5, disk version 2.0
I: FS at SDMMC: is 31116288 0x200-byte blocks with 512 cycle
I: sizes: rd 512 ; pr 512 ; ca 512 ; la 2048
/SDMMC:: bsize = 512 ; frsize = 512 ; blocks = 31116288 ; bfree = 31116284

Listing dir /SDMMC: ...
[FILE] boot_count (size = 1)
[FILE] pattern.bin (size = 547)
E: file open error (-12)
E: FAIL: open /SDMMC:/boot_count: -12
I: /SDMMC: unmounted
/SDMMC: unmount: 0

2. CONFIG_MAIN_STACK_SIZE=2048

Without this, I get stack overflow
*** Booting Zephyr OS build zephyr-v3.5.0-500-g57dd116d8b32 ***
Sample program to r/w files on littlefs
I: LittleFS version 2.5, disk version 2.0
I: FS at SDMMC: is 31116288 0x200-byte blocks with 512 cycle
I: sizes: rd 512 ; pr 512 ; ca 512 ; la 2048
E: ***** MPU FAULT *****
E:   Stacking error (context area might be not valid)
E:   Data Access Violation
E:   MMFAR Address: 0x240029f8
E: r0/a1:  0x98c055ce  r1/a2:  0x44001c95  r2/a3:  0xbb769fcc
E: r3/a4:  0xebeccbc6 r12/ip:  0x480e0624 r14/lr:  0x6e6673d5
E:  xpsr:  0x9ff8cc00
E: Faulting instruction address (r15/pc): 0x947d8070
E: >>> ZEPHYR FATAL ERROR 2: Stack overflow on CPU 0
E: Current thread: 0x240004c8 (unknown)
E: Halting system
E: disk interface already registered!!
I: littlefs partition at /lfsemmc